xds/internal/balancer/lrs/lrs.go

```diff
@@ -44,7 +46,7 @@ type Store interface {
 	CallFinished(l internal.Locality, err error)
 	CallServerLoad(l internal.Locality, name string, d float64)
 	// Report the load of clusterName to cc.
```

Comment:
This comment probably needs some updating.

Reply:
This comment is still right. The new hostname isn't part of the load, it's more like part of the node.

If the caller can add it to the node, the extra parameter won't be necessary (Hmm, this actually sounds better, I will see if it's doable).

Reply:
I made the change. Now the `Store` interface is unchanged. Metadata is added by the caller. PTAL.